examples/cra/src/pages/cowShedHook/index.tsx

```diff
@@ -0,0 +1,96 @@
+import '../../pageStyles.css'
+import { FormEvent, useCallback, useEffect, useState } from 'react'
+import { CowShedHooks } from '@cowprotocol/cow-sdk'
+import { useCurrentChainId } from 'hooks/useCurrentChainId'
+import { useWeb3Info } from 'hooks/useWeb3Info'
+import { JsonContent } from '../../components/jsonContent'
+import { ResultContent } from '../../components/resultContent'
+import { formatBytes32String } from 'ethers/lib/utils.js'
+import { SigningScheme } from '@cowprotocol/contracts'
+
+const DEADLINE = BigInt(1000000)
```

Comment:
just to make it easier to read
```suggestion
const DEADLINE = BigInt(1_000_000)
```